classes/sushi/CounterR5Report.inc.php

```diff
@@ -0,0 +1,537 @@
+<?php
+
+/**
+* @file classes/sushi/CounterR5Report.inc.php
+*
+* Copyright (c) 2013-2021 Simon Fraser University
```

Comment:
(Copyright dates should be bumped for all new code)

Reply:
What does this mean?

Reply:
Change the copyright end dates to 2022. (Actually for new code it should just be (c) 2022, but if the start date is older, it's probably because the header got copied from another file -- which is I suppose derivative :))